backend/src/api/tasks/tasks.api.ts

```diff
@@ -99,6 +103,45 @@ const initTasksApi: FastifyPluginAsync<Options> = async (fastify, opts) => {
       rep.status(HttpCode.OK).send(notes);
     },
   });
+
+  fastify.route({
```

Comment:
Does our swagger documentation include these routes?

Reply:
Gonna add them after the daily, forgot about them :eyes: